ca/ca.go

```diff
@@ -764,8 +779,18 @@ func (ca *CertificateAuthorityImpl) integrateOrphan() error {
 	if err != nil {
 		return fmt.Errorf("failed to parse orphan: %s", err)
 	}
-	issued := cert.NotBefore.Add(-ca.backdate)
-	_, err = ca.sa.AddCertificate(context.Background(), orphan.DER, orphan.RegID, orphan.OCSPResp, &issued)
+	issued := cert.NotBefore.Add(ca.backdate)
```

Comment:
I'm not sure I understand why this was changed to `ca.backdate` instead of `-ca.backdate`.

In `generateSerialNumberAndValidity` the `ca.backdate` is inverted when being added to the base `notBefore`:
https://github.com/letsencrypt/boulder/blob/93902965e5cda788a1cdf99593abee5310b34fd8/ca/ca.go#L549

Reply:
Right. `generateSerialNumberAndValidity` is backdating the cert. `integrateOrphan` is un-backdating the cert, trying to conclude the actual issued timestamp from the configured backdate. It's the reverse operation so the sign is reversed. I'll add a comment.

Reply:
Ahhhh! Ok, now I understand :+1: